core/workspace_svg.js

```diff
@@ -916,6 +917,19 @@ Blockly.WorkspaceSvg.prototype.showContextMenu_ = function(e) {
     }
     Blockly.Events.setGroup(false);
   }
+
+  var deleteOption = {
+    text: deleteList.length == 1 ? Blockly.Msg.DELETE_BLOCK :
```

Comment:
Is this also? I remember changing this code before. No problem though, we can fix it back after the merge.